rust/datafusion/src/execution/context.rs

```diff
@@ -259,6 +260,11 @@ impl ExecutionContext {
         self.datasources.insert(name.to_string(), provider);
     }
 
+    /// Get a reference to the registered data sources
```

Comment:
Since we already have a `table` method for accessing a registered data source, I wonder if it would be better to add a method to get a list of registered table names:

```rust
pub fn tables(&self) -> Result<Vec<String>>
```

This would avoid exposing the internal hash map.

Reply:
Ok, i use tables instead of datasources